images/inventory-generator/root/usr/local/bin/user_setup

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+set -x
+
+# ensure $HOME exists and is accessible by group 0 (we don't know what the runtime UID will be)
+chown ${USER_UID}:0 ${HOME}
+chmod ug+rwx -R ${HOME}
+
+# runtime user will need to be able to self-insert in /etc/passwd
```

Comment:
Ah, will word that comment a bit better. This comment justifies the need to make `/etc/passwd` writable, in order to tie a username to the given uid. This is required for ssh to work from within the container